swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/profiles/:id':
+    put:
+      description: 'Controller: security. Action: createOrReplaceProfile.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/profiles/:id/_createOrReplace':
+    put:
+      description: 'Controller: security. Action: createOrReplaceProfile.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_create:
+    post:
+      description: 'Controller: security. Action: createProfile.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_mget:
+    post:
+      description: 'Controller: security. Action: mGetProfiles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /profiles/_search:
+    post:
+      description: 'Controller: security. Action: searchProfiles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/roles/:id':
+    put:
+      description: 'Controller: security. Action: createOrReplaceRole.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/roles/:id/_createOrReplace':
+    put:
+      description: 'Controller: security. Action: createOrReplaceRole.'
+      parameters:
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /roles/_create:
+    post:
+      description: 'Controller: security. Action: createRole.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /roles/_mget:
+    post:
+      description: 'Controller: security. Action: mGetRoles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  /roles/_search:
+    post:
+      description: 'Controller: security. Action: searchRoles.'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/users/:id':
```

Comment:
missing also methods for route '/users/:id':
* "get" => security/getUser
* "post" => security/updateUser
* "delete" => security/deleteUser